hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.hadoop.hdds.scm.client;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
remove unused import

```suggestion
```

Reply:
If we remove the annotation, then it does become unused indeed.